index.html

```diff
@@ -91,17 +137,51 @@ <h3 class="program">Second Round</h3>
                 <li>
                     <div class="final-session">
                         <div>
-                            <img src="images/final annoucement.png" alt="">
+                            <img src="images/finish-line.png" alt="">
                         </div>
                         <h3 class="program">Final Session</h3>
                         <p class="program-description">Do not forget to grab your bonuses for 2024 festival!</p>
                     </div>
                 </li>
             </ul>
-            <a class="see" href="https://www.capitaljazz.com/supercruise/2023/">SEE THE WHOLE PROGRAM</a>
+            <div class="see-it">
+                <a class="see" href="https://www.capitaljazz.com/supercruise/2023/">SEE THE WHOLE PROGRAM</a>
+            </div>
         </div>
     </section>
-    <script href="script.js"></script>
+    <div class="art">
+        <h2 class="artists-heading">Featured Artists</h2>
+        <hr class="separator2" />
+    </div>
+    <article class="singers-card"></article>
+    <article class="singers-card1"></article>
+    <article class="singers-card2"></article>
+    <footer class="footer">
+        <section class="footer-card" id="partners">
+            <div class="f-title">
+                <p class="footer-title">
+                    Partners
+                </p>
+            </div>
+            <hr class="strikethrough">
+            <div class="partners" id="sponsor">
+                <div class="footer-list">iHeart</div>
+                <div class="footer-list">Spotify</div>
+                <div class="footer-list">MTV</div>
+                <div class="footer-list">Billboard</div>
+                <div class="footer-list">Vogue</div>
+            </div>
```

Comment:
- [ ] Please add the logos of your supporters as requested by the previous reviewer. 🥺